src/Util/EMailer/MailBuilder.php

```diff
@@ -159,15 +160,32 @@ public function withRecipient(string $address)
 	}
 
 	/**
-	 * Adds new headers to the default headers
+	 * Adds a value to a header
+	 *
+	 * @param string $name The header name
+	 * @param string $value The value of the header to add
+	 *
+	 * @return static
+	 */
+	public function addHeader(string $name, string $value)
+	{
+		$this->headers[$name][] = $value;
+
+		return $this;
+	}
+
+	/**
+	 * Sets a value to a header (overwrites existing values)
 	 *
-	 * @param string $headers New headers
+	 * @param string $name The header name
+	 * @param string $value The value to set
 	 *
 	 * @return static
 	 */
-	public function addHeaders(string $headers)
+	public function	setHeader(string $name, string $value)
 	{
-		$this->headers .= $headers;
+		$this->headers[$name] = [];
+		$this->headers[$name][] = $value;
```

Comment:
```suggestion
		$this->headers[$name] = [$value];
```

Reply:
Is it really the same? Adding a value to an values array?